backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -56,19 +56,19 @@ export class AdminResolver {
       searchText,
       currentPage = 1,
       pageSize = 25,
-      notActivated = null,
-      isDeleted = null,
+      filterByActivated = null,
```

Comment:
so we not use the name convention `withFilter` but `filterByFilter`? If so please adjust the other Call aswell

Reply:
I created a follow up issue: #1840